app/data/repositories/DecksRepository.py

```diff
@@ -0,0 +1,66 @@
+from app.data.repositories.UsersRepository import user
+from app.data import dbConn
+from app.data.models.Deck import Deck
+
+
+conn = dbConn.get()
+
+
+def create(user_id, name):
```

Comment:
define that it returns bool

Reply:
I think c.lastrowid returns int or None, still should I define returning bool?

Reply:
I think yes, because you are using what that method return value to know if create method ended with success or not. So false -> no success, true -> success. None will be casted to false, non 0 int will be casted to true.